src/BlockTypes/ProductCollection.php

```diff
@@ -4,6 +4,10 @@
 
 use WP_Query;
 
+// phpcs:disable WordPress.DB.SlowDBQuery.slow_db_query_tax_query
+// phpcs:disable WordPress.DB.SlowDBQuery.slow_db_query_meta_query
+// phpcs:disable WordPress.DB.SlowDBQuery.slow_db_query_meta_key
```

Comment:
Nit: in general, I think it's better to disable linting on a per-line basis instead of doing it for the entire file. It makes it more intentional, specially with these specific rules, so it's easy to see which queries might be the slow ones.

Reply:
Agree! Fixed! 19ff45e5b02af0d2c126a75b01272ebfa30f9a21